WordPress/Classes/ViewRelated/Comments/Controllers/Composer/CommentComposerReplyCommentView.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+import WordPressUI
+
+final class CommentComposerReplyCommentView: UIView, UITableViewDataSource {
+    private let tableView = UITableView(frame: .zero, style: .plain)
+    private let comment: Comment
+    private let helper = ReaderCommentsHelper()
+    private lazy var heightConstraints = tableView.heightAnchor.constraint(equalToConstant: 120)
+
+    init(comment: Comment) {
+        self.comment = comment
+
+        super.init(frame: .zero)
+
+        addSubview(tableView)
+        tableView.pinEdges()
+
+        tableView.separatorStyle = .none
+        tableView.alwaysBounceVertical = false
+        tableView.register(CommentContentTableViewCell.defaultNib, forCellReuseIdentifier: CommentContentTableViewCell.defaultReuseID)
```

Comment:
Is resuing `CommentContentTableViewCell` the only reason for using a table view here? If that's the case, maybe it's worthwhile to refactor `CommentContentTableViewCell` to extract its content into a plain view, and the cell itself becomes just a shell/wrapper.

Reply:
Ideally I'd do this. I did this with new `ReaderPostCell`. But it wasn't something I could prioritize at the time. I'm also planning to remove the xib

>  the only reason for using a table view here?

Technically it's also needed for scrolling. If not for scrolling, I would've definitely prioritized refactoring now.